docs/user_guide/queue_config.md

```diff
@@ -265,7 +271,19 @@ The star "*" is the wildcard character and matches all users or groups.
 Duplicate entries in the lists are ignored and do not cause a parsing error.
 Specifying a star beside other list elements is not allowed.
 When a wildcard group is configured, a limit must be configured with at least one named group.
-Parsing will reject the configuration with limits that do not follow this rule.
+
+Valid User name:
+
+* must start with lower case or upper case alphabet,
+* followed by alphanumeric character ([a-z0-9A-Z]), underscores (_), colons (:), hashes (#), slashes (/), ampersands (&), dots (.), and dashes (-).
```

Comment:
Ditto, no "ampersands (&)", should it be @?

> var UserRegExp = regexp.MustCompile(`^[_a-zA-Z][a-zA-Z0-9:#/_.@-]*[$]?$`)

([Link](https://github.com/apache/yunikorn-core/blob/6939b13d1d0e65aef18722ef546413f3e1775be2/pkg/common/configs/configvalidator.go#L77))